frontend/test/pytest/test_jax_scipy_functions.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests to check JAX SciPy functionality with qjit
+"""
+from jax import numpy as jnp
+from jax import scipy as jsp
+import numpy as np
+import pytest
+
+from catalyst import qjit, jacobian
+from catalyst.utils.exceptions import CompileError
+
+matrix_inputs = [
+    jnp.array([[0.1, 0.2], [5.3, 1.2]]),
+    jnp.array([[1, 2], [3, 4]]),
+    jnp.array([[1.0, -1.0j], [1.0j, -1.0]]),
```

Comment:
wanna add `jnp.array([[1,2,3],[4,5,6],[3,2,1]])`? This is the matrix that failed in https://github.com/PennyLaneAI/catalyst/issues/1071

Reply:
```suggestion
    jnp.array([[1.0, -1.0j], [1.0j, -1.0]]),
    jnp.array([[1, 2, 3],[4, 5, 6],[3, 2, 1]]),
```